pulp_rpm/app/models/repository.py

```diff
@@ -148,6 +151,61 @@ class RpmRemote(Remote):
     """
 
     TYPE = 'rpm'
+    sles_auth_token = models.CharField(
+        max_length=512,
+        null=True,
+        default=None
+    )
+
+    @property
+    def download_factory(self):
+        """
+        Return the DownloaderFactory which can be used to generate asyncio capable downloaders.
+
+        Upon first access, the DownloaderFactory is instantiated and saved internally.
+        Plugin writers are expected to override when additional configuration of the
+        DownloaderFactory is needed.
```

Comment:
Is it a relevant comment?

Reply:
good catch, it is not relevant, refactor leftover